apis/Google.Cloud.Compute.V1/Google.Cloud.Compute.V1/NodeTypesClient.g.cs

```diff
@@ -571,23 +575,57 @@ public override NodeType Get(GetNodeTypeRequest request, gaxgrpc::CallSettings c
         /// </summary>
         /// <param name="request">The request object containing all of the parameters for the API call.</param>
         /// <param name="callSettings">If not null, applies overrides to this RPC call.</param>
-        /// <returns>The RPC response.</returns>
-        public override NodeTypeList List(ListNodeTypesRequest request, gaxgrpc::CallSettings callSettings = null)
+        /// <returns>A pageable sequence of <see cref="NodeType"/> resources.</returns>
+        public override gax::PagedEnumerable<NodeTypeList, NodeType> List(ListNodeTypesRequest request, gaxgrpc::CallSettings callSettings = null)
         {
             Modify_ListNodeTypesRequest(ref request, ref callSettings);
-            return _callList.Sync(request, callSettings);
+            return new gaxgrpc::GrpcPagedEnumerable<ListNodeTypesRequest, NodeTypeList, NodeType>(_callList, request, callSettings);
         }
 
         /// <summary>
         /// Retrieves a list of node types available to the specified project.
         /// </summary>
         /// <param name="request">The request object containing all of the parameters for the API call.</param>
         /// <param name="callSettings">If not null, applies overrides to this RPC call.</param>
-        /// <returns>A Task containing the RPC response.</returns>
-        public override stt::Task<NodeTypeList> ListAsync(ListNodeTypesRequest request, gaxgrpc::CallSettings callSettings = null)
+        /// <returns>A pageable asynchronous sequence of <see cref="NodeType"/> resources.</returns>
+        public override gax::PagedAsyncEnumerable<NodeTypeList, NodeType> ListAsync(ListNodeTypesRequest request, gaxgrpc::CallSettings callSettings = null)
         {
             Modify_ListNodeTypesRequest(ref request, ref callSettings);
-            return _callList.Async(request, callSettings);
+            return new gaxgrpc::GrpcPagedAsyncEnumerable<ListNodeTypesRequest, NodeTypeList, NodeType>(_callList, request, callSettings);
         }
     }
+
+    public partial class AggregatedListNodeTypesRequest : gaxgrpc::IPageRequest
+    {
+        /// <inheritdoc/>
+        public int PageSize
```

Comment:
As noted on the generator PR, I think this would be better using explicit interface implementation:

```csharp
gaxgrpc::IPageRequest int PageSize
```

That way it doesn't show up as publicly - it's odd to have a public set-only property. If we want to actually have a public property, it's probably worth including a getter as well.

Reply:
I've updated the corresponding comment on the generator PR (https://github.com/googleapis/gapic-generator-csharp/pull/344) to keep the discussion in one place